tests/h/traversal/roots_test.py

```diff
@@ -312,12 +313,14 @@ def test_it_does_not_assign_create_permission_without_user_role(self, pyramid_co
 
         assert not pyramid_request.has_permission('create', context)
 
-    def test_it_returns_the_group_if_it_exists(self, factories, group_factory):
+    def test_getitem_returns_fetched_group_if_not_None(self, factories, group_factory, group_service):
         group = factories.Group()
+        group_service.fetch.return_value = group
 
         assert group_factory[group.pubid] == group
```

Comment:
It looks like the `group` object may be unnecessary in this test. Would `assert group_factory[group.pubid] == group_service.fetch.return_value` work?